content/en/getting_started/application_security/_index.md

```diff
@@ -0,0 +1,96 @@
+
+   You don't need to examine every suspicious request individually. ASM generates _signals_ when your attention is required, which you can learn about in Phase 2.
+
+4. Now that ASM is set up detecting suspicious requests on your service in real-time, **let the product run for a couple days**.
+
+<div class="alert alert-info">If your services use recent versions of tracing libraries, ASM immediately <a href="/security/application_security/risk_management">detects security vulnerabilities</a> in the upstream libraries that your services have as dependencies, and APM previews this information for you in its Security view. Optionally, <strong>explore ASM's consolidated <a href="https://app.datadoghq.com/security/appsec/vm">Vulnerabilities view</a></strong> (beta) to see how it helps you triage and prioritize vulnerabilities.</div>
+
+**Note**: ASM also enables you to **track user activity**, identifying authenticated users making suspicious requests. [Setting up authenticated user tracking][5] is not required to use ASM, but it does help you get visibility on authenticated attacks and attackers. 
+
+## Phase 2: Review your first security signals and vulnerabilities
+
+After a few days of usage, you typically get your first security signals. If you have not received your first security signal, you can simulate an attack and trigger a signal by running the following script, customized to hit your application's endpoints:
+
+{{< code-block lang="sh" >}}
+for ((i=1;i<=200;i++)); do
+# Target existing service’s routes
+curl https://your-application-url/<EXISTING ROUTE> -A
+'dd-test-scanner-log';
+# Target non existing service’s routes
+curl https://your-application-url/<NON-EXISTING ROUTE> -A
+'dd-test-scanner-log';
+done{{< /code-block >}}
+
+Over the next few days:
+
+1. **Review security signals at [Security --> Application Security --> Signals][6].** Security signals are generated by ASM when your attention is required. The signal severity provides you with insights on how quickly you need to react. 
+   
+   Aim to review `INFO` and `LOW` signals once a week, `MEDIUM` and `HIGH` once a day, and `CRITICAL` signals immediately.
+
+2. **Click a signal to see its details.** The signal details tell you what happened, who the attackers are, and what you should do next. On the Traces tab, you can investigate the traces that generated the signals.
+
+3. **Archive the signal after it's been investigated.** If the signal is a false positive, set up an [exclusion filter][9] to eliminate noisy signal patterns.
+
+4. **Set up a [notification][12]** to alert you in real-time when attacks require your attention, over email, Slack, or other communication integration. We recommend setting a [notification rule][11] for `Medium` and higher severity signals.
+
+5. **[Subscribe to the weekly digest][10]** to receive weekly reports on your security signals.
```

Comment:
Delete; included in 4

Reply:
These are very different activities from a customer perspective, and so deserve separate steps in this checklist.